manifest.yml

```diff
@@ -17,4 +17,4 @@ applications:
     services:
     - voice-of-interconnect-cloudant
     - voice-of-interconnect-speech-to-text
-    - voice-of-interconnect-natural-language-understanding
```

Comment:
I had to create a separate `manifest.yml` file to make the "Deploy to Bluemix" button work. There existed already a `manifest.prod.yml` and `manifest.staging.yml` files, but no `manifest.yml` file. The problem was that those files get transformed into a `manifest.yml` file by Travis CI, but there is no Travis CI step when using the "Deploy to Bluemix" button.

Since I was creating a separate `manifest.yml` file anyways, I decided to clean up the service instance names:

* Added a `voice-of-interconnect-cloudant` (not in the other manifest files)
* Renamed `Speech to Text-19` to `voice-of-interconnect-speech-to-text`
* Added `voice-of-interconnect-natural-language-understand` (not in the other manifest files) and then later renamed it to `voice-of-interconnect-natural-language` as the other name was too long.

The service instance names are all prefaced with `voice-of-interconnect-` because service instances are scoped to spaces in Bluemix, not to specific apps.

I'd be happy to clean up the service instance names in `manifest.prod.yml` and `manifest.staging.yml` as well, but that should be a separate pull request and needs to be done in coordination with changing the service instance names in Bluemix so we don't break the production app.